applications/nrf5340_audio/dfu/empty_net/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+#
```

Comment:
Why is the empty NET required?

Reply:
For generating NET bootloader and signature.